updated reference to Autofac 3.5.2 and Nunit 2.6.4

gcaferra opened the pull request:
please accept this PR our build system claim Autofac 3.3 and fails to build.

Thanks

Commit:
updated reference to Autofac 3.5.2 and Nunit 2.6.4

Commit:
modified the version to reflect the autofac version

tillig commented:
You probably need to add [an assembly binding redirect](https://msdn.microsoft.com/en-us/library/eftw1fys%28v=vs.110%29.aspx) to your app.config or web.config file to point to the new version if you're updating. Sometimes NuGet misses adding them. We don't want to issue a new NuGet package with artificially increased minimum requirements because it can inadvertently force folks to upgrade who don't need to.

gcaferra commented:
I'm sorry before rebuilding I tried with the assembly redirect  to do that and I also removed and added all references again and didn't work. the only way to solve our problem is rebuild with the given version.
Can you can add this in next release?

tillig commented:
We have a policy of only updating the minimum package requirements if they truly change. The next release will probably be 4.0 across the board.

We see these sorts of pull requests _a lot_. In the end, truly, there's always a way to change it with a redirect or other configuration option. It can sometimes be tricky to find the right spot to put the redirect to fix the issue, but generally it's possible. Sorry I can't be of more help.
